tests/test_handle_error.py

```diff
@@ -0,0 +1,86 @@
+import unittest
+from unittest.mock import Mock
+import asyncio
+import argparse
+import aiohttp
+import shutil
+import os
+from snare import HttpRequestHandler
+
+
+class AsyncMock(Mock):  # custom function defined to mock asyncio coroutines
```

Comment:
if you are going to use this class multiple times, it's better to make a separate class (I would like to suggest you to make something like utils module)